src/firefly/js/ui/ToolbarButton.jsx

```diff
@@ -48,13 +48,15 @@ function getShortCutInfo(shortcutKey) {
 export const ToolbarButton = memo((props) => {
     const {
         icon,text='',tip='',badgeCount=0,enabled=true, visible=true,
-        imageStyle={}, shortcutKey='', color='neutral',
-        disableHiding, active, sx,
+        imageStyle={}, shortcutKey='', color='neutral', variant='plain',
+        disableHiding, active, sx:inSX,
```

Comment:
This is a ToolbarButton, you would not expect `sx` to apply to `Tooltip` element.  If given, it should apply to the root.  I think slotProps should be used instead.

Reply:
now that I think about it there is a way to do what I want with out modifying the passed sx.